torch/_deploy.py

```diff
@@ -50,17 +47,11 @@ def persistent_load(saved_id):
         return serialized_storages[data[0]]
 
 
-    import_module : Callable[[str], ModuleType] = importlib.import_module
+    importer: Importer
     if zip_reader is not None:
-        importer = _get_package(zip_reader)
+        importer = OrderedImporter(_get_package(zip_reader), sys_importer)
```

Comment:
Will this cause issues if there isn't a zip_reader? `Importer` has `import_module` defined as an abstract method if I'm looking at the right files. Should it be set to default to `_SysImporter`?

Reply:
good catch, will fix